docs/ATOMIC_DESIGN.md

```diff
@@ -0,0 +1,21 @@
+# Atomic Design System
+
+This document describes the structure of the `src/lib/components` folder for creating new Components
+
+## Overview
+
+Welcome to our Atomic Design System! This system is built on the principles of Atomic Design, which organizes user interfaces into discrete levels of complexity—Atoms, Molecules, Organisms, and beyond.
```

Comment:
I have the feeling that we are not exactly using atomic design here? Does that matter?

Reply:
@josue2591 you are right, I just updated it